lib/smart_answer_flows/coronavirus-employee-risk-assessment.rb

```diff
@@ -127,42 +134,43 @@ def define
         option :yes
         option :no
 
-        next_node do |response|
-          if response == "yes"
-            outcome :vulnerable_work_arrangements
-          else
-            question :do_you_live_with_someone_vulnerable?
-          end
+        on_response do |response|
+          calculator.are_you_vulnerable = response
+        end
+
+        next_node do
+          question :do_you_live_with_someone_vulnerable?
         end
       end
 
       multiple_choice :do_you_live_with_someone_vulnerable? do
         option :yes
         option :no
 
-        next_node do |response|
-          if response == "yes"
-            outcome :keep_your_household_safe
-          else
-            question :have_childcare_responsibility?
-          end
+        on_response do |response|
+          calculator.do_you_live_with_someone_vulnerable = response
+        end
+
+        next_node do
+          question :have_childcare_responsibility?
         end
       end
 
       multiple_choice :have_childcare_responsibility? do
         option :yes
         option :no
 
-        next_node do |response|
-          if response == "yes"
-            outcome :help_with_childcare
-          else
-            outcome :go_back_to_work
-          end
+        on_response do |response|
+          calculator.have_childcare_responsibility = response
+        end
+
+        next_node do
+          outcome :go_back_to_work
         end
       end
 
       # Outcomes
+      outcome :you_should_not_be_going_to_work
       outcome :work_from_home
       outcome :work_from_home_help
       outcome :workplace_should_be_closed
```

Comment:
I think the follow outcomes aren't used any more:
-  workplace_should_be_closed
- vulnerable_work_arrangements
- keep_your_household_safe
- help_with_childcare

Should we remove them?

Reply:
Yh I was just doing this now - updated the PR.

Thanks for flagging